e2e-tests/pom.xml

```diff
@@ -21,6 +21,7 @@
         <module>spring-graphql</module>
         <module>spring-rest-mysql</module>
         <module>spring-rpc</module>
+        <module>wiremock</module>
```

Comment:
no need to create its own module. should be just a new package under `spring-rest-openapi-v2` (if writing in Java) or `v3` (if in Kotlin). see for example `e2e-tests/spring-rest-openapi -v2/src/test/java/org/evomaster/e2etests/spring/examples/db`